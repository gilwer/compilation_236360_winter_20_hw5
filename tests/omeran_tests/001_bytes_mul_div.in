// bytes and mul/div
void main() {
     byte x = 2b;
     printi(x); // 2
     byte y = 3b;
     printi(y); // 3
     y = y / x;
     printi(y); // 1
     int z = x * 7;
     printi(z); // 14
     int w = z / 3b;
     printi(w); // 4

     byte zero;
     printi(zero); // 0
 }