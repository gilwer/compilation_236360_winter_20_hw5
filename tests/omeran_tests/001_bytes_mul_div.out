2
3
1
14
4
0
